cms/static/js/spec/views/previous_video_upload_list_spec.js

```diff
@@ -99,6 +107,121 @@ define(
                 // Verify that nothing changes when we click on Cancel button on confirmation popup
                 verifyVideosInfo(this, false);
             });
+
+            describe('Paging Previous Video Upload List View', function() {
+
+                var videoView
+
+                beforeEach(function() {
+                    var collection = new VideoPagingCollection([
```

Comment:
I would rename collection to something more meaningful. VideosCollection maybe .